lib/stat_tracker.rb

```diff
@@ -544,4 +544,82 @@ def most_goals_scored(team_id)
     highest_goal
   end
 
+
+  def fewest_goals_scored(team_id)
+    lowest_goal = 100000
```

Comment:
I wonder if there is a cleaner way to implement lowest_goal. I agree 100000 is entirely unattainable, I'm wondering if there is maybe a more readable way to do this? I'll look it up!